.travis.yml

```diff
@@ -6,14 +6,20 @@ before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
   - sudo apt-get install -qq g++-4.8 libarmadillo-dev libarmadillo4 libboost1.54-all-dev
+  - curl http://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+  - sudo add-apt-repository "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-4.0 main"
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq clang-4.0 clang-tidy-4.0
```

Comment:
Is clang-tidy-3.8 acceptable?  That's already available in the trusty repositories.